blogposts/more-intuitive-code-search-on-sourcegraph-dot-com.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Search the top 10,000 open source repositories on Sourcegraph.com"
+description: "How Sourcegraph.com became easier to use, by setting the default search scope to the top ten thousand open source repositories on GitHub."
```

Comment:
minor nit: can we be consistent about "10,000" and "ten thousand"? (I think 10,000 pops more)

Reply:
Ok, will change to be 10,000.